### ISSUE TITLE
Thunderbird signed + encrypted message: "..." instead of actual subject

### ISSUE THREAD

**Issue by limonte** [FlowCrypt/flowcrypt-browser#3943]
While working on #3941 I sent 3 types of messages to `ci.tests.gmail@flowcrypt.dev`:

- signed html (all good)
- signed text (all good)
- signed + encrypted html (faced issues)

Here's how it looks like in Thunderbird:

![CleanShot 2021-08-30 at 18 40 24@2x](https://user-images.githubusercontent.com/6059356/131365843-58d88b39-6cd2-4d17-9b42-1d795f8b5fbb.png)

And here's how it looks in FlowCrypt:

![CleanShot 2021-08-30 at 18 41 05@2x](https://user-images.githubusercontent.com/6059356/131365942-3f2e1a69-beba-41a3-815c-6eb864fcd973.png)

**Comment by tomholub** [comment]
This used to at least render the subject inside the message body, but I'm not seeing it on your screenshot. I wonder why.

This code:

```ts
      if (decoded.subject && isEncrypted && (!plainSubject || !Mime.subjectWithoutPrefixes(plainSubject).includes(Mime.subjectWithoutPrefixes(decoded.subject)))) {
        // there is an encrypted subject + (either there is no plain subject or the plain subject does not contain what's in the encrypted subject)
        decryptedContent = this.getEncryptedSubjectText(decoded.subject, isHtml) + decryptedContent; // render encrypted subject in message
      }

```

**Comment by tomholub** [comment]
Thank you for noticing

**Comment by limonte** [comment]
The issue is happening when this option is on (it's on by default when the user clicks "Require encryption"):

![CleanShot 2021-09-22 at 19 20 11@2x](https://user-images.githubusercontent.com/6059356/134382944-3e7c95c2-1df9-4305-87be-8eb268595566.png)

**Comment by tomholub** [comment]
Yup. My comment above should still hold - we have code in place to deal with this, but it no longer works. Presumably, after Thunderbird was refactored this year, they send the subject in a bit different format and we no longer recognize it.
